### PR TITLE
Adding a new file

### PR DESCRIPTION
**Pull request by mykey06**
closes #43

### CODE REVIEW

File: mykey06.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+## Recommendations when visiting JPL
+
+### Places to Eat
+
```
Comment (alexandye) [inline mykey06.md:5]: there's no places to eat :(

File: mykey06.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
...
+### To Do
```
Comment (alexandye) [inline mykey06.md:6]: there's no things to do!

**Review by alexandye — APPROVED** [review]
there's nothing here!

File: mykey06.md (hunk 1 above)
Comment (mykey06) [reply mykey06.md:5]: There's nothing to eat here.

File: mykey06.md (hunk 2 above)
Comment (mykey06) [reply mykey06.md:6]: you can go anywhere!

**Pull request merged**